.github/workflows/cd.yml

```diff
@@ -2,6 +2,7 @@ name: cd
 on:
   workflow_run:
     workflows: ["ci"]
+    branches-ignore: [master]
```

Comment:
So we do not want to run this workflow after CI has finished successfully? That's what I was trying to do on #829

Reply:
yeah we don't need to run it on master.